meta-resin-common/recipes-containers/docker/docker_git.bb

```diff
@@ -113,7 +113,7 @@ do_compile() {
   export CGO_LDFLAGS="${LDFLAGS}  ${TARGET_CC_ARCH} --sysroot=${STAGING_DIR_TARGET}"
 
   DOCKER_GITCOMMIT="${SRCREV}" \
-    ./hack/make.sh dynbinary
+    ./hack/make.sh binary-rce
```

Comment:
Can't we make this be one line? Perhaps it's broken because it used to be longer.